Changelog.md

```diff
@@ -5,6 +5,9 @@ Changelog
 
 * Use ceil in `ResettingController` for a better token lifetime approximation
 * Removed unused translation keys
+* Removed form deprecations
+* Added `symfony/templating` as a dependency
```

Comment:
I don't think this is important enough. It was made because FrameworkBundle removed it (btw, the next change means it can probably be removed again)